### PR TITLE
Fix the render command to not wipe ca.pem

### LINKED ISSUE #1003: render credentials sometimes wipes ca.pem on v0.9.9-rc.2
Steps:
* `init` a new cluster render
* Copy `ca.pem` and `ca-key.pem` into the credentials folder for this new cluster
* Run `render credentials`, all creds generated as expected
* Run `render credentials` again, for example after changing some LB config
* `ca.pem` is now empty

### PR DESCRIPTION
**Pull request by mumoshu**
Fixes #1003 

@c-knowles Would this work for you?

### CODE REVIEW

**Comment by mumoshu** [comment]
Squashed a fix for an easy mistake. Please reload to see the latest commit id if anyone has already seen it.

File: core/controlplane/config/encrypted_assets.go [hunk 1]
```diff
@@ -440,39 +440,46 @@ func ReadOrEncryptAssets(dirname string, manageCertificates bool, caKeyRequiredO
...
+	}
+
 	assets := []struct {
-		name      string
-		data      []byte
-		overwrite bool
+		name             string
+		data             []byte
+		overwrite        bool
+		ifEmptySymlinkTo string
```
Comment (c-knowles) [inline core/controlplane/config/encrypted_assets.go]: Does this mean if populated, create a symlink?

**Comment by mumoshu** [comment]
Squashed again. Seems to work so far.

File: core/controlplane/config/encrypted_assets.go (hunk 1 above)
Comment (mumoshu) [reply core/controlplane/config/encrypted_assets.go]: If we had nothing to write, default to create a symlink to denoted by the `ifEmptySymlinkTo`.
Basically, we have nothing to write for `worker-ca.pem` as long as the user explicitly overwrite or put the file before running `kube-aws render credentials`. So in that case, create a symlink to `ca.pem`(=the same as the previous behavior).

**Comment by mumoshu** [comment]
I've manually verified this to work by running `kube-aws render` to regenerate credentials and then updating my cluster with `kube-aws update`.
The cfn stack update was successful while kube-dashboard went into CrashLookBackOff probably due to the rotation of the service account signing key.

```
$ k logs kubernetes-dashboard-7f46664f8f-n7gm4
2017/11/21 13:50:43 Starting overwatch
2017/11/21 13:50:43 Using in-cluster config to connect to apiserver
2017/11/21 13:50:43 Using service account token for csrf signing
2017/11/21 13:50:43 No request provided. Skipping authorization
2017/11/21 13:50:43 Error while initializing connection to Kubernetes apiserver. This most likely means that the cluster is misconfigured (e.g., it has invalid apiserver certificates or service accounts configuration) or the --apiserver-host param points to a server that does not exist. Reason: the server has asked for the client to provide credentials
Refer to the troubleshooting guide for more information: https://github.com/kubernetes/dashboard/blob/master/docs/user-guide/troubleshooting.md
```

However, this issue is not yet resolved in kube-aws - see #1013 for more info.

**Comment by mumoshu** [comment]
I'll merge this once someone put LGTM on it

**Pull request merged**